tempesta_fw/http_parser.c

```diff
@@ -2717,10 +2667,16 @@ tfw_http_parse_resp(void *resp_data, unsigned char *data, size_t len)
 
 	__FSM_START(parser->state) {
 
+	/* ----------------    EOL    ---------------- */
+
+	#define TFW_HTTP_RESP_EOL
+	#include "http_parser_eol.h"
+	#undef TFW_HTTP_RESP_EOL
+
```

Comment:
Do we really need this in a separate file? I would say it should be in this file.